packages/zowe-explorer/src/trees/job/JobFSProvider.ts

```diff
@@ -225,18 +227,23 @@ export class JobFSProvider extends BaseProvider implements vscode.FileSystemProv
         await AuthHandler.waitForUnlock(spoolEntry.metadata.profile);
         const query = new URLSearchParams(uri.query);
         let recordRange = "";
+        const recordsToFetch = SettingsConfig.getDirectValue<number>("zowe.jobs.paginate.recordsToFetch") ?? 0;
+
         if (query.has("startLine")) {
             const startLine = parseInt(query.get("startLine")!);
-            const endLine = query.has("endLine")
-                ? (parseInt(query.get("endLine")!))
-                : (startLine + (SettingsConfig.getDirectValue<number>("zowe.jobs.paginate.recordsToFetch") ?? 0)) - 1;
+            const endLine = query.has("endLine") ? parseInt(query.get("endLine")!) : startLine + (recordsToFetch - 1);
```

Comment:
Would the same problem also occur here when records to fetch is 1?
I know its highly unlikely that someone would use this number for that setting, but if `recordsToFetch` is 1 here, then `endLine` ends up being the same value as `startLine` rather than `startLine + 1`